program.c

```diff
@@ -0,0 +1,78 @@
+#include<stdio.h>
+#include<stdlib.h>
+#include<ctype.h>
+#include<string.h>
+int isOperator(char ch){
+    if(ch=='+' || ch=='-' || ch=='/' || ch=='*')
+        return 1;
+    else    
+        return 0;
+}
+int solve(int a,int b,char op){
+    if (op=='+') 
+        return a+b;
+    if (op=='-') 
+        return a-b;
+    if (op=='*') 
+        return a*b;
+    if (op=='/'){
+        if(b==0){
+            printf("Error: Division by zero.\n");
+            exit(0);
+        }
+        return a/b;
+    }
+    return 0;
+}
+int evaluate_expression(char* expression){
+    int n=strlen(expression);
+    int i=0;
+    int numbers[100];
+    char operators[100];
+    int num_top=-1;
+    int op_top=-1;
+    while(i<n){
+        if(isspace(expression[i])){

```

Comment:
Usage of inbuilt functions like **isspace()** is not allowed

Reply:
all inbuilt function are removed in calculator branch.